sum_square_difference30.py

```diff
@@ -0,0 +1,39 @@
+# The sum of the squares of the first ten natural numbers is, 1² + 2² + 3²...+ 10² = 385
+
+# The square of the sum of the first ten natural numbers is, (1+2+3...+10)²=55²= 3025
+
+# Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is 
+# 3025 - 385 = 2640.
+
+# Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum
+
+
+
+def first_ten():
+    first = 0
+    tots = 0
+    res = 0
+    for x in range(101):
+        first += x ** 2
+        sums = (first)
+
+
+        # the first for loop will gather the 1² + 2² + 3²...+ 10² = 385 or wichever number is wanted 
```

Comment:
nice, except this comment is probably describing line 16-18, so it should go directly above in line 15, also this is too much extra whitespace xP